semgrep/semgrep/cli.py

```diff
@@ -540,6 +540,5 @@ def cli() -> None:
                 timeout_threshold=args.timeout_threshold,
                 skip_unknown_extensions=args.skip_unknown_extensions,
                 severity=args.severity,
-                report_time=output_time,
```

Comment:
There were a lot of changes and I lost track a bit---how is `self.settings.output_time` in output.py being set? It controls whether or not the timing summary is outputted when we're using the text output.

Reply:
It is set in cli.py depending on whether --time or --json-time is passed